content/sensu-go/5.19/reference/checks.md

```diff
@@ -479,6 +479,10 @@ Learn how to use check hooks with the [Sensu hooks reference documentation][6].
 
 ## Check specification
 
+{{% notice note %}}
+**NOTE**: The Sensu Go check definition does not include the `occurrences` attribute. In Sensu Go, [occurrences are included in events][60].
```

Comment:
Occurrences are also tracked in Sensu Classic events, so the distinction implied here isn't very meaningful to users.

Classic had no concept of metadata, instead arbitrary key/value pairs could be added directly along side other attributes which now go in the `spec` block. `occurrences` in the check definition started out as arbitrary but became a convention.

Reply:
Thank you -- I think I phrased that poorly. Does my update make more sense?